iocore/net/UnixNet.cc

```diff
@@ -70,20 +71,30 @@ class InactivityCop : public Continuation
         continue;
       }
 
+      if (ne->default_inactivity_timeout_in == -1) {
+        // If no context-specific default inactivity timeout has been set by an
+        // override plugin, then use the global default.
+        Debug("inactivity_cop",
+              "vc: %p setting a the global default inactivity timeout of %d, next_inactivity_timeout_at: %" PRId64, ne,
+              nh.config.default_inactivity_timeout, ne->next_inactivity_timeout_at);
+        ne->set_default_inactivity_timeout(HRTIME_SECONDS(nh.config.default_inactivity_timeout));
```

Comment:
Not sure when `initialize_thread_for_net` is called relative to when configs get changed but why don't we use `global_config` var, which reads the config values as opposed to `nh->config` which is just a copy that can possibly have stale data? Is it guaranteed to be updated when configs change?

Reply:
That's an interesting idea, but I think we better stick with reading the config from the nethandler here rather than from global_config. Currently global_config is populated into each handler with each handler registering the `update_nethandler_config` for updates that I presume gets processed in a timely fashion. All the configurations are looked at in this file from nethandler, not directly from global_config. That is, notice that global_config is only used for nethandler config initialization and then subsequent updates via the update handler, it's never used to access the runtime values. I'm not sure, but this may be due to threading coordination reasons.

In any case, I'd say we stick with what is done in the rest of this file for this PR. If we want to look into changing things in UnixNet.cc to access global_config directly, let's do that in a separate PR.